blueprints/addon/files/.travis.yml

```diff
@@ -3,9 +3,14 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - "4"
+  - "6"
```

Comment:
We had intentionally left this at 4 so that folks would be testing against their lowest supported node. If we do update this, we need to change/remove the comment directly above.